backend/src/main/kotlin/ch/yvu/teststore/run/overview/RunOverviewService.kt

```diff
@@ -17,6 +17,10 @@ open class RunOverviewService @Autowired constructor(
         open val resultRepository: ResultRepository,
         val resultDiffService: ResultDiffService) {
 
+    fun getRunOverviewById(runId: UUID): Optional<RunOverview> =
+            Optional.ofNullable(runRepository.findById(runId))
```

Comment:
I have no clue why I used `Optional` instead of nullable types in the other methods. This seems like unnecessary overhead. If you want to you can refactor the Services to return nullable types instead of Optionals (e.g. `RunOverview?` instead of `Optional<RunOverview`).